tutorials/plot_ecog.py

```diff
@@ -0,0 +1,47 @@
+"""
+======================
+Working with ECoG data
+======================
```

Comment:
It already is. You seem to be looking at / linking to the `stable` docs but look instead at the CircleCI build:

https://2987-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/tutorials.html